src/client/components/Profile.jsx

```diff
@@ -0,0 +1,84 @@
+import { use } from "bcrypt/promises";
+import { useState, useEffect } from "react";
+
+const userDummyData = {
+    id: 1,
+    username: "NathanTheSamosa",
+    password: "my-secret",
+    email: "nathanthesamosa@gmail.com",
+    admin: false
+}
+
+const userDummyId = userDummyData.id
+
+const URL = process.env.REACT_APP_API_URL;
+const postEndpoint = `/post/user/${userDummyId}`;
+const postURL = URL + postEndpoint;
+
+
+
+
+const Profile = () => {
+
+    const [posts, setPosts] = useState(null)
```

Comment:
This should probably initialise to an empty array or you'll encounter problems with `posts.map` trying to run .map on `null` during initial render

Reply:
Noted. Thanks Nathan.